s3cortxutils/s3confstore/s3confstore/cortx_s3_confstore.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import argparse
+# from cortx.utils.conf_store import Conf
+from cortx.utils.conf_store import ConfStore
+from urllib.parse import urlparse
+import os.path
+import json
+
+conf_store = ConfStore()
+
+def load_config(index, backend_url):
+  """Instantiate and Load Config into constore"""
+  conf_store.load(index, backend_url)
+  return conf_store
+
+
+class S3CortxConfStore:
+
+  def __init__(self):
+    print("constructor()")
+
+  def load(self, index, path):
+    print("in subroutine load()")
+    result_data = load_config(index, path)
+
+  def test_load(self, index, path):
```

Comment:
Also I think we can make the get option as part of this PR... rename the subroutine name from load_test to get_key().

We can have both load and get as part of this PR.

Reply:
Added getkey() as discussed.